examples/ppo/remote.py

```diff
@@ -1,16 +1,20 @@
+"""Utilities for running the agents in separate processes."""
+
 import multiprocessing
 import numpy as onp
-from collections import namedtuple
-from env import create_env
+import collections
+
+import env_utils
 
-exp_tuple = namedtuple('exp_tuple',
-                  ['state', 'action', 'reward', 'value', 'log_prob', 'done'])
+exp_tuple = collections.namedtuple(
```

Comment:
This is actually a class, so naming should be CamelCase

Reply:
Changed as part of https://github.com/google/flax/pull/470/commits/50b2b792454832469eeeb38189352a5f53071f65, thanks for spotting!